Formula/fastme.rb

```diff
@@ -0,0 +1,35 @@
+class Fastme < Formula
+  desc "Accurate and fast distance-based phylogeny inference program"
+  homepage "http://www.atgc-montpellier.fr/fastme/"
+  url "https://gite.lirmm.fr/atgc/FastME/raw/master/tarball/fastme-2.1.5.2.tar.gz"
+  sha256 "834e1f555b6057b23650cb214c1f2f899978d72d085fca863fb3f03f2bbf00fc"
+
+  option "with-openmp", "Enable parallel computations with OpenMP"
+
+  if build.with? "openmp"
+    depends_on "gcc" 
```

Comment:
There's some whitespace on this line, after the closing `"`.

Reply:
Sorry about that. Audit now passes.